<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelSetMainWindow</class>
 <widget class="QMainWindow" name="LabelSetMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>730</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>LabelSet</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
	border-radius: 5px;
	border: 0.6px solid black;
}
QPushButton:disabled {
	border: 0.6px solid gray;
	color: gray;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="imageView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>400</width>
      <height>640</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0.6px solid black</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="backImgSet">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>back</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextImgSet">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>340</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>next</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="labelSetInput">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelSetLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>210</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Label Set:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="labelSetSave">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>210</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>save</string>
    </property>
   </widget>
   <widget class="QLabel" name="datasetSelectLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Dataset:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="datasetSelectDirPathShow">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="datasetSelect">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>50</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportDatasetJson">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>600</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>export</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imageIndexView">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>340</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="sepLabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>340</y>
      <width>20</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>/ </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="imageListLengthLabel">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>340</y>
      <width>40</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="dataJsonLabel">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Dataset Json:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="datasetJsonFilePathView">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>280</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="datasetJsonImport">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>130</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>import</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unlabeledView">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>600</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>info</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageSizeView">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>660</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
